### PR TITLE
Fixes nodes.TagAttributes.__iter__

### PR DESCRIPTION
**Pull request by funkyfuture**

### CODE REVIEW

File: _delb/nodes.py [hunk 1]
```diff
@@ -645,7 +645,7 @@ def __getitem__(self, item: Union[str, slice]) -> Optional[Attribute]:
             return None
 
     def __iter__(self) -> Iterator[str]:
-        yield from self._element.attrib  # type: ignore
+        return iter(self._element.attrib)  # type: ignore
```
Comment (JKatzwinkel) [inline _delb/nodes.py:648]: why tho? i don't really see why it would need fixing..?

**Review by JKatzwinkel — APPROVED** [review]

File: _delb/nodes.py (hunk 1 above)
Comment (JKatzwinkel) [reply _delb/nodes.py:648]: I mean i can iterate over a `TagNode`'s `.attributes` as is, so how is this a bug?

Comment (funkyfuture) [reply _delb/nodes.py:648]: the [https://docs.python.org/3/reference/datamodel.html#object.__iter__](method) shall return an [iterator](https://docs.python.org/3/glossary.html#term-iterator) object, and these need to implement an `__iter__` method as well which afaict isn't the case yet.

Comment (JKatzwinkel) [reply _delb/nodes.py:648]: alright

**Pull request merged**